src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck start work");
+    }
+
+    @Override
+    public void stopWork() {
+
+        System.out.println("Truck stop work");
+

```

Comment:
redundant empty lines, let's remove them

Reply:
Do I need to make a new pull request?

Reply:
no, you need to make fixes in this PP(pull request)